### DISCUSSION TITLE
Trying to detect change in ScrolledText

### DISCUSSION THREAD

**Discussion by Cromwell1963** [israel-dryer/ttkbootstrap#407] [General]
I am trying to respond to changes in the text of a ScrolledText widget.
I have tried:

`self.scrolledtext.bind("<<TextModified>>", lambda: print("OK"))`

and:

`self.scrolledtext.text.bind("<<TextModified>>", lambda: print("OK"))`

but cant get either to work.
Id be grateful for suggestions.
Thanks

**Comment by israel-dryer** [comment]
I believe the event you are looking for is `<<Modified>>`
https://tcl.tk/man/tcl8.6/TkCmd/event.htm#M46

This would be bound to the `scrolledtext.text` object.

**Reply by Cromwell1963** [reply]
Thanks for the suggestion. When I do that, the event only triggers when I first load the ScrolledText widget. Any editing I do subsequently fails to fire the event. Should I be using a different virtual event for editing? thanks

......

I came back to it later and got it working.
Not sure what was wrong but it works now. thanks
